include/mppic/optimizer.hpp

```diff
@@ -17,18 +18,22 @@
 
 #include "mppic/motion_models.hpp"
 #include "mppic/critic_manager.hpp"
-#include "mppic/tensor_wrappers/control_sequence.hpp"
-#include "mppic/tensor_wrappers/state.hpp"
+#include "mppic/models/control_sequence.hpp"
+#include "mppic/models/control_constraints.hpp"
+#include "mppic/models/state.hpp"
+#include "mppic/models/sampling_std.hpp"
 #include "mppic/utils.hpp"
 
 namespace mppi
 {
 
+using StampType = decltype(std::declval<std_msgs::msg::Header>().stamp);
```

Comment:
This is a very odd line. Why not use the actual type? https://github.com/ros2/common_interfaces/blob/master/std_msgs/msg/Header.msg#L6

Reply:
Oh, sure, i'll make changes in next PR